app/views/shared/_organization_header.html.erb

```diff
@@ -13,14 +13,18 @@
         <span class="lead"><%= t('.marc_code', code: @organization.code) %></span>
         <% end %>
         <% if @organization.contact_emails.any? %>
-        <span class="lead"><%= mail_to @organization.contact_emails.first, t('.contact') %></span>
+        <span class="lead"><%= mail_to @organization.contact_emails.first.email, t('.contact') %></span>
         <% end %>
       </div>
       <div class="ms-auto">
-        <% if current_page?(organization_path(@organization)) %>
-        <%= link_to can?(:edit, @organization) ? t('.manage_org') : t('.view_org'), organization_users_path(@organization), class: 'align-self-center' %>
+        <% if current_page?(organization_path(@organization)) ||
```

Comment:
I think with some partial re-organization you could simplify the logic here with a `yield`

`<%= content_for :org_header_section_link %>`